src/manufacturer/manufacturerDialog.component.tsx

```diff
@@ -10,79 +14,51 @@ import {
   TextField,
   Typography,
 } from '@mui/material';
-
-import React from 'react';
-
 import {
   AddManufacturer,
-  Manufacturer,
   EditManufacturer,
   ErrorParsing,
-  ManufacturerDetails,
+  Manufacturer,
 } from '../app.types';
-import { useAddManufacturer, useEditManufacturer } from '../api/manufacturer';
 import { AxiosError } from 'axios';
+import { useAddManufacturer, useEditManufacturer } from '../api/manufacturer';
 import handleIMS_APIError from '../handleIMS_APIError';
-import { trimStringValues } from '../utils';
+
+const manufacturerSchema = z.object({
```

Comment:
The values sent are all empty strings except URL which is undefined. Ideally they would all be undefined or null when not given as the database schema is specifically either null or a value.
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/663d8ac2-adc1-46c5-8985-afc766c40e54)